schemas/core/SystemSettings.json

```diff
@@ -117,6 +117,14 @@
       "fieldtype": "Data",
       "readOnly": true,
       "hidden": true
+    },
+    {
+      "fieldname": "darkMode",
+      "label": "Dark mode",
+      "fieldtype": "Check",
+      "default": "false",
```

Comment:
`false` shouldn't be enclosed in quotes, it should be written as a boolean variable.
```json
"default": false,
```